docs/products/postgresql/howto/migrate-aiven-db-migrate.rst

```diff
@@ -1,32 +1,38 @@
-Migrate to Aiven for PostgreSQL with ``aiven-db-migrate``
-=========================================================
+Migrating to Aiven for PostgreSQL with ``aiven-db-migrate``
```

Comment:
We've standardised on imperative form for titles rather than the gerund (yes, we need a style guide!)